packages/next-international/src/types.ts

```diff
@@ -9,3 +10,50 @@ export type LocaleContext<Locale extends BaseLocale> = {
 export type LocaleMap<T> = Record<keyof T, React.ReactNode>;
 
 export type ReactParamsObject<Value extends LocaleValue> = Record<Params<Value>[number], React.ReactNode>;
+
+export type I18NProviderConfig = {
```

Comment:
Let's match the convention and write `I18n` instead of `I18N`:
```suggestion
export type I18nProviderConfig = {
```